cmd/tempo/app/modules.go

```diff
@@ -422,10 +423,15 @@ func (t *App) initQueryFrontend() (services.Service, error) {
 	// http endpoint to see usage stats data
 	t.Server.HTTPRouter().Handle(addHTTPAPIPrefix(&t.cfg, api.PathUsageStats), usageStatsHandler(t.cfg.UsageReport))
 
+	t.Server.HTTPRouter().PathPrefix("/static/").HandlerFunc(http.FileServer(http.FS(staticFiles)).ServeHTTP).Methods("GET")
+
 	// todo: queryFrontend should implement service.Service and take the cortex frontend a submodule
 	return t.frontend, nil
 }
 
+//go:embed static
+var staticFiles embed.FS
```

Comment:
this is blowing my mind. how are the files  in ./static mapped here?

Reply:
I know, me too.  With this PR http requests at `/static` will return files in the `./static` directory, which are stored in this variable at compile time.

Reply:
oh! 

```
//go:embed static
```

"static" means the ./static folder. my goodness